lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -30,7 +32,7 @@
                     this.teamsDetailUrl = options.teamParams.teamsDetailUrl;
                     this.action = options.action;
 
-                    _.bindAll(this, 'cancelAndGoBack', 'createOrUpdateTeam');
+                    _.bindAll(this, 'cancelAndGoBack', 'createOrUpdateTeam', 'deleteTeam', 'editMembership');
```

Comment:
I know you didn't introduce this, but I'm pretty sure it's not necessary -- Backbone takes care of binding event handlers.